lib/fromRdf.js

```diff
@@ -225,22 +225,23 @@ api.fromRDF = async (
       }
 
       // the list is nested in another list
-      if(property === RDF_FIRST) {
-        // empty list
-        if(node['@id'] === RDF_NIL) {
-          // can't convert rdf:nil to a @list object because it would
-          // result in a list of lists which isn't supported
-          continue;
-        }
-
-        // preserve list head
-        if(RDF_REST in graphObject[head['@id']]) {
-          head = graphObject[head['@id']][RDF_REST][0];
-        }
-
-        list.pop();
-        listNodes.pop();
-      }
+      // FIXME
```

Comment:
@gkellogg I think this is another case of trying to adapt the ruby changes. Is this block needed anymore?

Reply:
My code has the same commented-out bit, and I don't recall the original motivation. I think the test suite is comprehensive here, so the code doesn't seem necessary in either library.

Reply:
We have adequate tests for list of lists in from Rdf, so this can be dropped.